commons/src/main/java/gov/cms/qpp/conversion/util/StringHelper.java

```diff
@@ -0,0 +1,95 @@
+package gov.cms.qpp.conversion.util;
+
+import java.util.Iterator;
+
+/**
+ * Helper class for custom code that isn't covered by Apache Commons or Google Guava.
+ */
+public class StringHelper {
+	/**
+	 * Joins a {@link Iterable} of {@link String}s together with the specified separator. Between the second to last and last
+	 * item, the specified conjunction is used instead.  This allows one to make a {@link String} like {@code Dog, Cow, and Moof}.
+	 *
+	 * @param iterable The object to join together
+	 * @param separator The {@link String} that splits most of the items in the iterable.
+	 * @param conjunction The {@link String} that splits the second to last and last item.
+	 * @return A joined {@link String}.
+	 */
+	public static String join(Iterable<String> iterable, String separator, String conjunction) {
```

Comment:
what's wrong with Collectors#joining (think String#join is a thing too?)

Reply:
None of them allow for a custom conjunction between the second-to-last and last element.  Sadly, there isn't a clean way to build these frameworks into `StringHelper#join` either without being inefficient or not allowing a generic `Collection` or `Iterable`.